### DISCUSSION TITLE
Optional Query parameter list

### DISCUSSION THREAD

**Discussion by jorgejesus** [fastapi/fastapi#7815] [Questions]
### First check

* [ x] I used the GitHub search to find a similar issue and didn't find it.
* [ x] I searched the FastAPI documentation, with the integrated search.
* [ x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

Documentation is very clear that we can use lists e,g 
```
@app.get("/")
async def my_query(q: List[str] = Query(["a","b","c","d"])):
```
How can I:
Have an ***optional*** list structure, from the documentation we include the key word `None` 
e.g:
```
values:str = None
```
Meaning we can have two scenarios:
- `/q=a&q=b (q is present and item is in list) ` 
- `/ (q is not present)`

Tryied: `Query(None,["a","b","c","d"])`  but it didnt worked

### Additional context
Using  `fastapi==0.49.0`

**Comment by phy25** [comment]
Not sure if you want the default to be `None` why you want `["a","b","c","d"]` to be a default as well. You can try `typing.Optional[typing.List] = Query(["a","b","c","d"])`.

**Comment by jorgejesus** [comment]
My apologies since we have the following note:

> Have in mind that in this case, FastAPI won't check the contents of the list.

Therefore there is no problem on missing the `q`  (and its content) on the query

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 ! :bowing_man: 

Thanks for reporting back and closing the issue @jorgejesus  :+1:
